embedded-bins/iptables/Dockerfile

```diff
@@ -2,7 +2,19 @@ ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
 RUN apk add build-base git file curl \
-	linux-headers pkgconf libnftnl-dev bison flex
+	linux-headers pkgconf bison flex
+
+# libmnl-dev: The libmnl.a is missing in libmnl-dev. Static build fails.
+# The issue is fixed in the last libmnl build, but it's not yet in alpine packages.
+# See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1010616
+# See: https://bugs.launchpad.net/ubuntu/+source/libmnl/+bug/1971523
+#
+RUN curl -L https://netfilter.org/projects/libmnl/files/libmnl-1.0.5.tar.bz2 \
+	| tar -C / -jx
+
+RUN cd /libmnl-1.0.5 && ./configure && make LDFLAGS=-all-static && make install
+
+RUN apk add libnftnl-dev
```

Comment:
I think should fix this in alpine

Reply:
actually, this might work: `apk add ... libmnl-dev libmnl-static`

Reply:
Yeah, it works, thanks!